### PR TITLE
Sharding Announcement

### PR DESCRIPTION
**Pull request by orta**
Anything missing? /cc @segiddins

### CODE REVIEW

File: _posts/2016-10-28-Sharding.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "CocoaPods Specs Repo Sharding"
+author: orta
+categories: cocoapods specs infrastructure
+---
+
+In March, CocoaPods hit an unfortunate problem with our [Specs Repo](http://blog.cocoapods.org/Master-Spec-Repo-Rate-Limiting-Post-Mortem/) being rate limited. 
+
+We announced that we were [planning on Sharding](http://blog.cocoapods.org/Master-Spec-Repo-Rate-Limiting-Post-Mortem/#too-many-directory-entries) the Specs Repo, but were hestitent to do it instantly. Now over six months later we are planning on running the scripts.
```
Comment (DanToml) [inline _posts/2016-10-28-Sharding.markdown]: /hestitent/hesitant

File: _posts/2016-10-28-Sharding.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- If you are using CocoaPods 1.x - you should not see any changes.
+- If you are using CocoaPods 0.39 and below, you will not see new pod versions from CocoaPods Trunk after we run the script until you upgrade to a 1.x version.
```
Comment (DanToml) [inline _posts/2016-10-28-Sharding.markdown]: We might need to check that 0.39 won't just break (without users manually checking out the pre-sharding hash, and running everything with `--no-repo-update`)

Comment (segiddins) [inline _posts/2016-10-28-Sharding.markdown]: It won't, it will raise an `Informative` saying the minimum version of the specs repo is higher

**Comment by orta** [comment]
I don't think I'll have much time over this weekend to wrap and ship this, can someone else wrap it ( feel free to change to your name if you want too )

As it'll raise an informative, can we add something like:

> If you want to use the last commit before we ran this script for backwards compatibility, you can add:

```ruby
source "https://github.com/cocoapods/Old-Specs.git"
```

> To the top of your Podfile.

and we can make a repo with 1 commit of all the old specs? Seems like not a lot of work on our side, and a lot of advantages for those still on older builds. e.g. archived projects

**Comment by segiddins** [comment]
👍🏻

**Comment by orta** [comment]
alright, updated 👍

**Pull request merged**